src/js/modules/meal_controller.js

```diff
@@ -1,4 +1,5 @@
 import LikesRequest from './network/requests/likes_request';
+import displayPopup from './popup';
```

Comment:
This was throwing an error when I try to run the start command on npm. 
`ERROR in ./src/js/modules/meal_controller.js 2:0-35
Module not found: Error: Can't resolve './popup' in '/Users/thangodrichard/microverse/capstone-js-project/src/js/modules'
resolve './popup' in '/Users/thangodrichard/microverse/capstone-js-project/src/js/modules'`